.github/workflows/regenerate_scheduled.yml

```diff
@@ -16,9 +16,9 @@ jobs:
       - name: Redownload, regenerate, reformat
         id: changes
         run: |
-          sudo apt-get update > /dev/null
-          sudo apt-get -qq upgrade > /dev/null
-          sudo apt-get -qq install binutils wget tar xmlstarlet flatpak > /dev/null
+          sudo apt-get -q update
+          sudo apt-get -q upgrade
+          sudo apt-get -q install binutils wget tar xmlstarlet flatpak
```

Comment:
oops, -qq also implied -y which now needs to be added separatly.

Also the manpage says:

> you should never use -qq without a no-action modifier such as -d, --print-uris or -s as
> APT may decide to do something you did not expect. Configuration Item: quiet.

Reply:
Fixed